### PR TITLE
[libharu] Fix renaming library when one of the build type is selected

### PR DESCRIPTION
**Pull request by JafarAbdi**
**Describe the pull request**
When only one of the build types is selected (Release or Debug) and `VCPKG_LIBRARY_LINKAGE` is `static`, building this package is failing due to trying to rename a non-existing file, this PR should fix this issue

```
CMake Error at ports/libharu/portfile.cmake:50 (file):
  file RENAME failed to rename

    /home/jafar/workspaces/vcpkg/packages/libharu_x64-linux/debug/lib/libhpdfs.a

  to

    /home/jafar/workspaces/vcpkg/packages/libharu_x64-linux/debug/lib/libhpdfd.a

  because: No such file or directory
```

### CODE REVIEW

File: ports/libharu/CONTROL [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Source: libharu
 Version: 2017-08-15-9
+Port-Version: 1
```
Comment (PhoebeHui) [inline ports/libharu/CONTROL:3]:
```suggestion
Version: 2017-08-15
Port-Version: 10
```

**Comment by PhoebeHui** [comment]
@JafarAbdi, thanks for the PR! Could you please update the version?

**Review by PhoebeHui — APPROVED** [review]

File: ports/libharu/portfile.cmake [hunk 2]
```diff
@@ -31,34 +31,41 @@ else()
...
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     if(NOT VCPKG_CMAKE_SYSTEM_NAME OR VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
-       file(RENAME ${CURRENT_PACKAGES_DIR}/lib/libhpdfs.lib ${CURRENT_PACKAGES_DIR}/lib/libhpdf.lib)
-       file(RENAME ${CURRENT_PACKAGES_DIR}/debug/lib/libhpdfsd.lib ${CURRENT_PACKAGES_DIR}/debug/lib/libhpdfd.lib)
+       if(NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
+          file(RENAME "${CURRENT_PACKAGES_DIR}/lib/libhpdfs.lib" "${CURRENT_PACKAGES_DIR}/lib/libhpdf.lib")
```
Comment (BillyONeal) [inline ports/libharu/portfile.cmake:46]: Renaming files after cmake install is... suspect, but the previous portfile was doing this too.

**Comment by BillyONeal** [comment]
Thanks :)

**Pull request merged**